### ISSUE TITLE
For T2V training, it happens that latent's size is torch.Size([1, 16, 21, 90, 135], but the size of noise_pred is torch.Size([1, 16, 21, 90, 134].

### ISSUE THREAD

**Issue by RockyLQ1** [modelscope/DiffSynth-Studio#441]
The mse_loss calculation encounters size mismatch problem. 
        loss = torch.nn.functional.mse_loss(noise_pred.float(), training_target.float())
which part may cause the issue？

**Comment by RockyLQ1** [comment]
For T2V task, it seems that setting the default size [480, 832] may run the project without mismatch in size, but not for 720p.
